### PR TITLE
Flow logs access

### PR DESCRIPTION
**Pull request by michaelwittig**

### CODE REVIEW

**Comment by michaelwittig** [comment]
test

File: module.yml [hunk 1]
```diff
@@ -354,6 +401,73 @@ Resources:
           Condition:
             StringEquals:
               's3:x-amz-acl': 'bucket-owner-full-control'
+  BucketPolicyCloudTrailCrossAccountWrite:
```
Comment (andreaswittig) [inline module.yml]: Is there a way to restrict which AWS account is able to write logs? For example, by specifying the account or OU id?

File: module.yml [hunk 2]
```diff
@@ -354,6 +401,73 @@ Resources:
...
+          Condition:
+            StringEquals:
+              's3:x-amz-acl': 'bucket-owner-full-control'
+        - Sid: AWSLogDeliveryAclCheck
+          Effect: Allow
+          Principal:
+            Service: 'delivery.logs.amazonaws.com'
+          Action: 's3:GetBucketAcl'
+          Resource: !GetAtt 'Bucket.Arn'
+  BucketPolicyFlowLogCrossAccountWrite:
```
Comment (andreaswittig) [inline module.yml]: Is there a way to restrict which AWS account is able to write logs? For example, by specifying the account or OU id?

**Comment by michaelwittig** [comment]
wait for AWS Support

**Comment by michaelwittig** [comment]
test

File: module.yml (hunk 1 above)
Comment (michaelwittig) [reply module.yml]: removed

File: module.yml (hunk 2 above)
Comment (michaelwittig) [reply module.yml]: removed

**Pull request merged**